packages/shared/src/schemas/websockets.mts

```diff
@@ -167,10 +167,15 @@ export const FileDeletedPayloadSchema = z.object({
   path: z.string(),
 });
 
-export const PreviewStatusPayloadSchema = z.object({
-  url: z.string().nullable(),
-  status: z.enum(['booting', 'running', 'stopped']),
-});
+export const PreviewStatusPayloadSchema = z.union([
+  z.object({ url: z.string().nullable(), status: z.enum(['booting', 'running']) }),
+  z.object({
+    url: z.string().nullable(),
+    status: z.literal('stopped'),
+    ok: z.boolean(),
```

Comment:
What's the semantic meaning of ok? Not immediately obvious to me, so let's leave a comment

Reply:
I'll change it to something more descriptive. To answer your question though, the intent was to store whether the process stopped successfully.